src/templates/template-engine-block/index.vue

```diff
@@ -502,6 +516,14 @@
     callbackUrl.value = uri
   }
 
+  const isHandleField = (field) => {
+    if (field === 'repository_name') {
+      if (hasIntegrations.value) return true
+      return false
+    }
+    return true
```

Comment:
if ( field !=='repository_name'){
return true
}

return hasIntegrations.value

Reply:
Ajuste acima para facilitar. Quando não é um repository_name, já retorna true. Já nos casos onde for igual, já retorna o valor do hasIntegrations.value ( que é um boolean)